ydb/docs/en/core/concepts/datamodel/_includes/table.md

```diff
@@ -134,7 +134,7 @@ If there are multiple followers, their delay from the leader may vary: although
 
 | Parameter name | Type | Acceptable values | Update capability | Reset capability |
 | ------------- | --- | ------------------- | --------------------- | ------------------ |
-| `TTL` | Expression | `Interval("<literal>") ON <column> [AS <unit>]` | Yes | Yes |
+| `TTL` | Expression | `<tier>, ..., <tier> ON <column> [AS <unit>]`, where `<tier>` — `Inteval("<literal>") TO EXTERNAL DATA SOURCE <path_to_source>` or `Inteval("<literal>") TO DELETE`. Short form:  `Interval("<literal>") ON <column> [AS <unit>]` — implies `DELETE` action | Yes | Yes |
```

Comment:
This became too complicated for a table cell. Instead, let's provide a link to a YQL reference article on this.